src/Mapbender/CoreBundle/Resources/public/sass/element/legend.scss

```diff
@@ -1,4 +1,10 @@
 
 // notshow -> hide always legend item
 // notvisible -> hide legend item by user
-.notshow, .notvisible{display:none;}
\ No newline at end of file
+.notshow, .notvisible{display:none;}
+
+.mb-element-legend ul li.image {
+
+  overflow: scroll;
```

Comment:
I would suggest "overflow: auto". It's just a small design tweak, but the scrollbars disappear if an element doesn't need them. That reduces the legend in many cases.
Alternatively, we could change the whole approach and change the image to a smaller scale than the sidepane-width (e.g. max-width: 200 px). Due to possible inconsistencies that can occur because of the size of different legend graphics and the adjustability of the sidepane, I don't quite know which solution fits more here.

Reply:
Of course, you are right on that. It needs to be "overflow: auto", my mistake.

Your second point is good also. This could be easily achieved by adding the bootstrap-css-class "img-responsive" to the <img>-elements. I tested this and it works. But the readability of some scaled down images may suffer from that, so I rather prefer using "overflow:auto"